backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -666,6 +666,7 @@ describe('UserResolver', () => {
   describe('logout', () => {
     describe('unauthenticated', () => {
       it('throws an error', async () => {
+        jest.clearAllMocks()
```

Comment:
What's the reason to clearAllMocks here?
There is no logger.test after the testcase.

Reply:
in case in some moments they happen to be logged in some way, it is already stated. 
It is just an order keeping thing, as it is a mock, there is no much computational strength used on clearing it